UR3eState state
UR3eTarget ball
UR3eTarget basket
---
UR3eTrajectoryRTA[] trajectories